### PR TITLE
Update lro version in package.json for autorest rlc generation

### PR DESCRIPTION
**Pull request by kazrael2119**
fixes https://github.com/Azure/autorest.typescript/issues/2760

### CODE REVIEW

**Comment by kazrael2119** [comment]
debug steps:
1. checkout this pr && rush update and build
2. in sdk repo, cd sdk\agrifood\agrifood-farming-rest\swaggers
3. run autorest --typescript ./readme.md --use= {your local codegen path} --typescript.debugger
4. open the url with edge or chrome: edge://inspect/#devices or chrome://inspect/#devices
5. wait the this browser catch the debugger and enter it, then add break point

**Comment by MaryGao** [comment]
Java's reference: https://github.com/Azure/autorest.java/blob/6fabb93051072929ea2641bcae63a3276fa7fd50/postprocessor/src/main/java/com/azure/autorest/postprocessor/Postprocessor.java#L133

File: packages/autorest.typescript/src/restLevelClient/generateRestLevel.ts [hunk 1]
```diff
@@ -121,6 +123,15 @@ export async function generateRestLevelClient() {
     generateFileByBuilder(project, buildPackageFile, rlcModels);
     // buildTsConfig
     generateFileByBuilder(project, buildTsConfig, rlcModels);
+  } else {
+    // update package.json if existing
+    const packageJsonContent = JSON.parse(await host.readFile("package.json"));
```
Comment (MaryGao) [inline packages/autorest.typescript/src/restLevelClient/generateRestLevel.ts:128]: @kazrael2119 it should work with readFile, could you have a verify?

**Review by MaryGao — APPROVED** [review]
LGTM

File: packages/autorest.typescript/src/restLevelClient/generateRestLevel.ts (hunk 1 above)
Comment (kazrael2119) [reply packages/autorest.typescript/src/restLevelClient/generateRestLevel.ts:128]: I have verified and this works well

**Pull request merged**